README.md

```diff
@@ -2,31 +2,200 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Introduce database management systems concepts
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Databases often contain massive amounts of data and support thousands
+of simultaneous users, and thus must be managed with sophisticated software tools.
 
-## Topic 1
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Relational Database Management Systems
 
-## Topic 2
+- **Data** is information that can be represented in many formats
+  including numeric, textual, visual, or audio.
+- A **database** is a collection of data in a structured format.
+- A **database management systems** or **DBMS** is software that reads and writes data
+  and often provides additional features including security, performance, transactions, multi-user access, and recovery. 
+- A **relational database management system** or **RDBMS** stores data in tables, columns and rows.
+  - A table holds information about objects of a similar type.
+  - Each column within a table has a unique name.
```

Comment:
While we're going over basic definitions, should we also state that a column in a database table is also referred to as a field and a row is a record?

Reply:
yes, thanks.